Sept11.sh

```diff
@@ -0,0 +1,22 @@
+#Script to setup a project directory and download a FASTA file
+#Alan Yu ayu15@dons.usfca.edu
+#September 11, 2017
+
+#set up directories
+echo "Setting up directories..."
+mkdir 2017-09-11_bioinformatics_project
+cd 2017-09-11_bioinformatics_project
+mkdir code data data/raw_data output output/figures output/tables
+echo "Done creating directories!"
+
+#download the FASTA file into the raw_data subdirectory
+echo "Downloading crab FASTA file.."
+cd data/raw_data
+curl -L http://npk.io/aMPCOe+ -o filename.fasta
```

Comment:
The intention was to have you choose a better name for the file than `filename.fasta`